### ISSUE TITLE
How to register and get API Key for ReKignition?

### ISSUE THREAD

**Issue by AmitAjmera** [orbeus/ReKognition_iOS_SDK#1]
Hello,

I am trying to integrate ReKognition ios sdk in my project but not aware about how to register and get "API Key" and "API Secret"?

Thanks

**Comment by YushanOrbeus** [comment]
Hi, 

Please get the key and secret on https://rekognition.com/

Best,
Yushan

**Comment by AmitAjmera** [comment]
Yes, I have already registered on this web but issue is that I am not getting verification Email.
 my Email id is: Amitajmera1990@gmail.com

Thanks

**Comment by tlyalex** [comment]
Hi Amit, 
  Our record shows that you have activated your account through our verification email.
  Please let us know if you are not able to use API. 
  Thanks.

**Comment by AmitAjmera** [comment]
Hello all,

Yes, My account activated successfully and It's working good now.

Thanks for your support.

Regards,
Amit Ajmera

**Comment by AmitAjmera** [comment]
Hello,

I am facing another issue, Result showing proper when I run app in device and simulator but Instead If I make distribution profile and Install it to my device then Results are not shown.

How do I resolve this?

Regards,
Amit Ajmera

**Comment by kh156** [comment]
Hi Amit,

The SDK should be provisioning-profile independent. Could you provide more details on the function calls you made and the results you got in each scenario?

Thanks,

**Comment by AmitAjmera** [comment]
Hello All,

I am getting proper result in Response for the features like Gender
Identity, Age etc. If I run in simulator or If I connect my device with
software but When I make Diawi link for particular device and install it
then It does not show any response.

I am using trial version, Can't I upload App with trial version of your SDK
to App store?

Regards,
Amit Ajmera
Skype: Amitajmera777
